src/components/ChartContainer/D3Container.js

```diff
@@ -22,6 +22,38 @@ function D3Container({ dashboardState, dashboardActions, store }) {
   const [byLineMeasure, setByLineMeasure] = useState('');
   const [byLineDisplayData, setByLineDisplayData] = useState([]);
 
+  const colorArray = [
+    "#003F5C",
+    "#2F4B7C",
+    "#665191",
+    "#A05195",
+    "#D45087",
+    "#F95D6A",
+    "#FF7C43",
+    "#FFA600",
+    "#9D02D7",
+    "#0000FF"
+  ]
+
+  const workingList = [];
+  store.results.forEach((item) => workingList.push(item.measure));
+  const measureList = Array.from(new Set(workingList));
+  const colorMap = measureList.map((item, index) => {
+    return ({
+      measure: item,
+      color: determineColor(index)
+    })
+  })
+
+  function determineColor(index) {
+    if (index <= 10) {
+      return colorArray[index]
+    }
+    else {
```

Comment:
`else` is redundant. Just return after the `if` block.

Reply:
Incorrect. The else in this statement accounts for any possible dataset that has over 10 entries by recycling the colors.

Reply:
No, `else` is redundant. The body of `else` is applied should `if` not be applied. Just `return colorArray[index % 10];`